di_website/publications/models.py

```diff
@@ -356,6 +356,16 @@ def meta_and_appendices(self):
         children += list(self.appendices)
         return filter(None, children)
 
+    @cached_property
+    def filtered_datasets(self):
```

Comment:
This block of code is repeated 6 times ... could you extract it and reuse, cut down on the lines of code.

Reply:
Made a mixin @edwinmp